sdk/attestation/azure-security-attestation/test/ut/policygetset_test.cpp

```diff
@@ -405,6 +405,7 @@ namespace Azure { namespace Security { namespace Attestation { namespace Test {
     std::string GetTestName(testing::TestParamInfo<PolicyTests::ParamType> const& testInfo)
     {
       std::string testName;
+      int suffixVotes = 0;
```

Comment:
If I understand the logic correctly, We only run ModifyPolicySecured on AAD in live mode.

Thus you could instead write the following code which is equivalent and easier to understand:
```
if(testInfo.param.InstanceType == ServiceInstanceType::AAD 
&& testInfo.param.TestType == TestCaseType::ModifyPolicySecured) {
    testName += "_LIVEONLY_";
}
```

Reply:
yes , the tests in attesteation are overengneered to the point where i couldn't find the code that it executes durin tests in order to debug ans see why i had issues running the code. I had to move it to liveonly. i filed a bug to Larry to look into this  when he gets back.